product/src/main/java/com/yas/product/service/ProductService.java

```diff
@@ -396,6 +396,9 @@ public void updateProduct(long productId, ProductPutVm productPutVm) {
         updateExistingVariants(productPutVm, allVariationImages, existingVariations);
         productRepository.saveAll(existingVariations);
 
+        Map<Long, ProductOption> optionsById = getProductOptionByIdMap(productPutVm.productOptionValues());
```

Comment:
Now that we have the `productOptionValueDisplay` field to store product option value information, could we remove the old `productOptionValues` field? The two types are quite similar, and using just one would eliminate the need to keep them consistent with each other.

Reply:
Hi @mochacr0, this part may be refactored to remove the productOptionValue, but it will be too complicate and make hard for review, we have to change a lot to remove it. So, I prefer to refactor it in another ticket.